moveit_ros/moveit_servo/src/utils/common.cpp

```diff
@@ -119,15 +119,38 @@ trajectory_msgs::msg::JointTrajectory composeTrajectoryMessage(const servo::Para
   joint_trajectory.header.frame_id = servo_params.planning_frame;
   joint_trajectory.joint_names = joint_state.joint_names;
 
-  trajectory_msgs::msg::JointTrajectoryPoint point;
+  static trajectory_msgs::msg::JointTrajectoryPoint point;
   point.time_from_start = rclcpp::Duration::from_seconds(servo_params.publish_period);
+  const size_t num_joints = joint_state.joint_names.size();
+  point.positions.resize(num_joints);
+  point.velocities.resize(num_joints);
+  point.accelerations.resize(num_joints);
```

Comment:
```suggestion
  const size_t num_joints = joint_state.joint_names.size();
  if (point.positions.size() != num_joints)
  {
    point.positions.resize(num_joints);
    point.velocities.resize(num_joints);
    point.accelerations.resize(num_joints);
  }
```

Reply:
I left out the check here so that the memory for the vectors are always pre-allocated rather than dynamically on "push_back"

Reply:
But the `point` variable is static now? I still think this suggestion makes sense.

Reply:
Ah! 
That makes sense , would only have to resize once since it is static unless joint group changes.